MAVProxy/modules/mavproxy_map/mp_tile.py

```diff
@@ -382,35 +371,30 @@ def load_tile(self, tile):
 		key = tile.key()
 		if key in self._tile_cache:
 			img = self._tile_cache[key]
-			if img == self._unavailable:
+			if np.array_equal(img, self._unavailable):
 				img = self.load_tile_lowres(tile)
 				if img is None:
 					img = self._unavailable
 				return img
 
 
 		path = self.tile_to_path(tile)
-		try:
-			ret = cv.LoadImage(path)
-
-                        # if it is an old tile, then try to refresh
-                        if os.path.getmtime(path) + self.refresh_age < time.time():
-                                try:
-                                        self._download_pending[key].refresh_time()
-                                except Exception:
-                                        self._download_pending[key] = tile
-                                self.start_download_thread()
+		ret = cv2.imread(path)
+		if ret is not None:
+			# if it is an old tile, then try to refresh
+			if os.path.getmtime(path) + self.refresh_age < time.time():
+				try:
+						self._download_pending[key].refresh_time()
+				except Exception:
+						self._download_pending[key] = tile
+				self.start_download_thread()
+					
 			# add it to the tile cache
 			self._tile_cache[key] = ret
 			while len(self._tile_cache) > self.cache_size:
 				self._tile_cache.popitem(0)
 			return ret
-		except IOError as e:
```

Comment:
Because to me the only reason for the try/catch block was due the fact that `cv.LoadImage` throws an IOError. In case of cv2 a non-existent image always delivered `None`, therefore I thought it would be ok to remove it. Do you think that there are other reasons?